src/js/lib/fileType.test.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+import path from "path"
+
+import lib from "./"
+
+const testFile = (name) => path.join(__dirname, "../../../itest/", name)
+const pcap = testFile("sample.pcap")
+const zeek = testFile("sample.zeek.log")
+const json = testFile("sample.json.zeek.log")
```

Comment:
Could you please make these come from sample.pcap already in the repo? You can do this by adding `itest/sample.pcap` to brim and then running the following on its bzng:
```sh
zq -f zeek itest/sample.pcap.brim/all.bzng > itest/sample.tsv
zq -f ndjson itest/sample.pcap/brim/all.bzng > itest/sample.ndjson
```
This makes it easier for testing and the resulting files will be smaller.

Reply:
Yes, I will do that.